examples/heterogenous_flow/example_h0002_het_show_gridpy

```diff
@@ -0,0 +1,41 @@
+# Copyright 2019 NREL
```

Comment:
This file needs a '.' for the extension.

Reply:
Fixed thanks!